src/lib/getNodeVersion.js

```diff
@@ -0,0 +1,15 @@
+const pkg = require('../../package.json');
+
+/**
+ * @example 14
+ * @returns {String} The minimum major Node.js version specified in the package.json
```

Comment:
Not sure if you want the min or max, but this code currently returns the latter:

```js
> Array.from("^12 || ^14 || ^16".matchAll(/\d+/g)).pop();
['16', index: 15, input: '^12 || ^14 || ^16', groups: undefined]
```

```suggestion
 * @returns {String} The maximum major Node.js version specified in the package.json
```

Reply:
Ooops thanks... yeah I wrote this comment when I was using `semver.minVersion` but ended up scrapping that since that'd require installing `semver` in the workflow 🙄